src/v0/destinations/sendgrid/deleteUsers.js

```diff
@@ -13,9 +12,38 @@ const {
 } = require("../../util/errorTypes");
 const tags = require("../../util/tags");
 
+/**
+ * This drops the user if userId is not available and converts the ids's into list of strings
+ * where each string is a combination of comma separated userIds and length of each string is not more than maxSize
+ * @param {*} userAttributes array of userIds
+ * @param {*} maxSize maxSize of url
+ * @returns list of Strings
+ */
+const chunksFromUrlLength = (userAttributes, maxSize) => {
+  const identity = [];
+  let left = maxSize;
+  let temp = "";
+  userAttributes.forEach(ua => {
+    // Dropping the user if userId is not present
+    if (ua.userId) {
+      left -= String(ua.userId).length;
```

Comment:
Shouldn't we include the comma `,` also in the length computation?

Reply:
Yes, We should. incorporating it.

Reply:
Changed the flow. Please check